ideascube/mediacenter/management/commands/import_medias.py

```diff
@@ -34,13 +38,26 @@ def abort(self, msg):
         self.stderr.write(msg)
         sys.exit(1)
 
-    def skip(self, msg, metadata=None):
-        self.stderr.write(u'⚠ Skipping. {}.'.format(msg))
-        if metadata:
-            for key, value in metadata.items():
-                value = value if value else ''
-                self.stdout.write(u'- {}: {}'.format(key, value))
-        self.stdout.write('-' * 20)
+    def report(self, name, item, level):
+        if level == 1:
+            self.errored = True
+        if name not in self._reports:
+            self._reports[name] = []
+        self._reports[name].append((item, level))
+
+    def reporting(self):
```

Comment:
rename this to `output_report` or `display_report`? In general it's easier to understand methods that have a verb in them.